rtdmultiprog.py

```diff
@@ -104,7 +104,8 @@ def isp_custom_instruction(cmd_type, cmd_code, read_n, write_n, write_value):
     # Execute custom instruction and wait until done
     write_reg(0x61, cmd_code)
     write_reg(0x60, (cmd_type<<5) | (write_n<<3) | (read_n<<1) | 1)
-    poll(lambda: not read_reg(0x60)[0] & 0x01, "Custom Instruction Timeout")
+    timeout = 20 if cmd_code == ERAS else 2
```

Comment:
```suggestion
    timeout = 20 if cmd_type == CI_ERASE else 2
```
I think it's better to check by command type rather than by opcode value.